Microsoft.Toolkit.Win32/Microsoft.Toolkit.Wpf.UI.Controls/InkToolbar/InkToolbar.cs

```diff
@@ -0,0 +1,287 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Markup;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Wpf.UI.Controls
+{
+    /// <summary>
+    /// Wpf-enabled wrapper for <see cref="Windows.UI.Xaml.Controls.InkToolbar"/>
+    /// </summary>
+    [ContentProperty(nameof(Children))]
+    public class InkToolbar : WindowsXamlHostBaseExt
+    {
+        internal Windows.UI.Xaml.Controls.InkToolbar UwpControl => ChildInternal as Windows.UI.Xaml.Controls.InkToolbar;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InkToolbar"/> class, a
+        /// Wpf-enabled wrapper for <see cref="Windows.UI.Xaml.Controls.InkToolbar"/>
+        /// </summary>
+        public InkToolbar()
+            : this(typeof(Windows.UI.Xaml.Controls.InkToolbar).FullName)
+        {
+            this.Visibility = System.Windows.Visibility.Collapsed;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InkToolbar"/> class, a
+        /// Wpf-enabled wrapper for <see cref="Windows.UI.Xaml.Controls.InkToolbar"/>.
+        /// </summary>
+        protected InkToolbar(string typeName)
+            : base(typeName)
+        {
+            Children = new ObservableCollection<DependencyObject>();
```

Comment:
When inherited, Visibility is not set to expected value `Collapsed` since that ctor is never called